packages/astro/src/env/sync.ts

```diff
@@ -1,32 +1,37 @@
-import fsMod from 'node:fs';
 import type { AstroSettings } from '../@types/astro.js';
-import { TYPES_TEMPLATE_URL } from './constants.js';
+import { ENV_TYPES_FILE } from './constants.js';
 import { getEnvFieldType } from './validators.js';
 
-export function syncAstroEnv(settings: AstroSettings, fs = fsMod): void {
-	if (!settings.config.experimental.env) {
-		return;
-	}
-
-	const schema = settings.config.experimental.env.schema ?? {};
-
-	let client = '';
-	let server = '';
+export function syncAstroEnv(settings: AstroSettings): void {
+	let client: string | null = null;
+	let server: string | null = null;
```

Comment:
Is there a reason to do `string | null = null` instead of `string = ''`? It seems to make the code below more complex.